ONSdigital/response-operations-ui
# Js bundling

# Motivation and Context
Lets us write modern, modularised code and bundle it for older browsers

# What has changed
Code split into modules
Bundler added
Pipeline and watcher added
Old bundler removed

# How to test?
Delete js in `static`.  run `npx gulp build` and confirm that a `main.min.js` file is created in `static/js`

# Links
https://trello.com/c/rlSmARte

# Screenshots (if appropriate):
